beacon-chain/core/electra/consolidations.go

```diff
@@ -262,7 +262,7 @@ func ProcessConsolidationRequests(ctx context.Context, st state.BeaconState, req
 		if !helpers.IsActiveValidator(srcV, curEpoch) || !helpers.IsActiveValidatorUsingTrie(tgtV, curEpoch) {
 			continue
 		}
-		// Neither validator are exiting.
+		// Neither validator is exiting.
```

Comment:
Nit as validators should also be plural then
```suggestion
		// Neither of the validators is exiting.
```